aur.in

```diff
@@ -20,6 +21,46 @@ if [[ -z $1 ]]; then
     done | column >&2
 
     exit 1
+}
+
+source /usr/share/makepkg/util/parseopts.sh
+
+opt_short=''
+opt_long=('color' 'no-color')
+opt_hidden=('dump-options')
+
+# broken hack to parse only up to first non-option.
+# has the side effect of only supporting long options
+# in the --long=arg form.
+
+declare -a aur_args
+for arg in $@; do
+    if [[ "$arg" != -* ]]; then
+        break
+    fi
+    aur_args+=("$arg")
+    shift
+done
+
+if ! parseopts "$opt_short" "${opt_long[@]}" "${opt_hidden[@]}" -- "${aur_args[@]}" -- "$@" ; then
+    usage
+fi
+set -- "${OPTRET[@]}"
+
+while true; do
+    case "$1" in
+        --color)        USE_COLOR=always ;;
+        --no-color)     USE_COLOR=never  ;;
```

Comment:
Shouldn't there be an `export` here?

Reply:
I exported the variable at the top of the file.
https://github.com/AladW/aurutils/blob/8f1cda7ca80235892a03b2cc18a20f19471b9045/aur.in#L5